Kitodo/src/main/webapp/WEB-INF/templates/includes/processFromTemplate/dialogs/fileUpload.xhtml

```diff
@@ -0,0 +1,75 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:f="http://xmlns.jcp.org/jsf/core"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:p="http://primefaces.org/ui">
+
+    <p:dialog widgetVar="fileUploadDialog"
+              id="fileUpload"
+              width="640"
+              modal="true"
+              resizable="false">
+        <h3>#{msgs['newProcess.fileUpload.heading']}</h3>
+        <h:form id="fileUploadForm">
+            <p:panelGrid layout="grid" columns="2" cellpadding="10">
+                <p:row>
+                    <div>
+                        <p:outputLabel for="catalogueSelectMenu"
+                                       value="#{msgs['newProcess.catalogueSearch.catalogue']}"/>
+                        <p:selectOneMenu id="catalogueSelectMenu"
+                                         autoWidth="false"
+                                         required="#{not empty param['catalogSearchForm:performCatalogSearch']}"
+                                         immediate="true"
+                                         value="#{CreateProcessForm.fileUploadDialog.selectedCatalog}">
+                            <f:selectItem itemValue="#{null}" itemLabel="-- #{msgs.selectCatalog} --" noSelectionOption="true"/>
+                            <f:selectItems value="#{CreateProcessForm.fileUploadDialog.catalogs}" var="step" itemLabel="#{step}"
+                                           itemValue="#{step}"/>
+                            <p:ajax update="fileUploadForm"/>
+                        </p:selectOneMenu>
+                    </div>
+                </p:row>
+            </p:panelGrid>
+            <p:panelGrid>
+                <p:row>
+                    <div>
+                        <p:fileUpload value="#{CreateProcessForm.fileUploadDialog.file}"
+                                      rendered="#{CreateProcessForm.fileUploadDialog.selectedCatalog ne null}"
```

Comment:
Because you already handle the file in the fileUploadListener and do not use the file stored to this value attribute `value` in the Java class, it should work without the `value` attribute entirely, I think.
```suggestion
                        <p:fileUpload rendered="#{CreateProcessForm.fileUploadDialog.selectedCatalog ne null}"
```